dev-apps/openfeature-nodejs/src/main.ts

```diff
@@ -61,6 +60,22 @@ async function startDevCycle() {
         metaDataFieldBoolean: true,
         metaDataFieldDouble: 1.23,
     })
+
+    const dvcUser = {
+        ...context,
+        user_id: context.targetingKey,
+    }
+    console.log(
```

Comment:
can we remove these since these are debug functions and instead highlight our `variable` or `variableValue` calls instead

Reply:
going to leave the `allFeatures()` call as I want an example of how to access the `devcycleClient` from the provider.